File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.js
function solution(order) {
    let answer = 0
    
    for (let menu of order) {
        if (menu.includes('americano') || menu === 'anything') {
            answer += 4500
        } else if (menu.includes('cafelatte')) {
            answer += 5000
        }     
    }
    
    return answer
}

File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.py
def solution(order):
    answer = 0
    
    for menu in order:
        if 'americano' in menu or menu == 'anything': # 핫/아이스 상관없이 '아메리카노'인 것(in 연산자 사용)과 '아무거나'일 때(== 연산자 사용)
            answer += 4500
        elif 'cafelatte' in menu: # 핫/아이스 상관없이 '카페라떼'일 때(in 연산자 사용)
            answer += 5000
    
    return answer
